pkg/controller/operators/catalog/operator.go

```diff
@@ -1331,10 +1331,44 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 		return
 	}
 
+	if plan.Status.Phase == v1alpha1.InstallPlanPhaseFailed {
+		return
+	}
+
 	querier := o.serviceAccountQuerier.NamespaceQuerier(plan.GetNamespace())
 	ref, err := querier()
 	if err != nil {
-		syncError = fmt.Errorf("attenuated service account query failed - %v", err)
+
+		// Retry sync if non-fatal error
+		if !scoped.IsOperatorGroupError(err) {
+			syncError = fmt.Errorf("attenuated service account query failed - %v", err)
+			return
+		}
+
+		// Mark the InstallPlan as failed for a fatal Operator Group related error
+		ipFailError := fmt.Errorf("attenuated service account query failed - %v", err)
```

Comment:
Right now, I'm propagating this error message to the InstallPlan status conditions. Wondering if this should be trimmed down a bit.
```Yaml
apiVersion: operators.coreos.com/v1alpha1
kind: InstallPlan
status:
  conditions:
    - lastTransitionTime: '2021-04-06T01:10:01Z'
      lastUpdateTime: '2021-04-06T01:10:01Z'
      message: >-
        attenuated service account query failed - no operator group found that
        is managing this namespace
      reason: InstallCheckFailed
      status: 'False'
      type: Installed
  phase: Failed
```

Reply:
Updated the status condition message to be more relevant to the user:
```YAML
apiVersion: operators.coreos.com/v1alpha1
kind: InstallPlan
status:
  conditions:
    - lastTransitionTime: '2021-04-06T01:10:01Z'
      lastUpdateTime: '2021-04-06T01:10:01Z'
      message: >-
        invalid operator group - no operator group found that
        is managing this namespace
      reason: InstallCheckFailed
      status: 'False'
      type: Installed
  phase: Failed
```